public/resources/ts/themes.ts

```diff
@@ -68,6 +68,9 @@ export function loadTheme(currentTheme: string): void {
           })
           .join(", ")})`;
         break;
+      case "linear-gradient":
+        value = 'linear-gradient(${background.angle}, ${background.color1} ${background.start}, ${background.color2} ${background.end})';
```

Comment:
literals should use ` not '
[MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)